pkg/client/cache/cache.go

```diff
@@ -23,10 +31,14 @@ func SaveToUserCache(ctx context.Context, object any, file string) error {
 	fullFilePath := filepath.Join(filelocation.AppUserCacheDir(ctx), file)
 	// get dir of joined path
 	dir := filepath.Dir(fullFilePath)
-	if err := dos.MkdirAll(ctx, dir, 0o700); err != nil {
+	var dirPerm fs.FileMode = 0o700
+	if perm == Public {
+		dirPerm = 0o755
+	}
+	if err := dos.MkdirAll(ctx, dir, dirPerm); err != nil {
```

Comment:
This is not going to work because we currently save private and public files in the same directory. The `apikeys.json` (saved from pro) must be private for instance. Other files in that directory are not private. We can't have a solution where the first saved file dictates the permissions of the directory.

Reply:
Okay; I'll make the directory readable then, since I don't want to implement the change to have a private directory we mount separately right now